tests/services/messenger/translators/discourse.spec.ts

```diff
@@ -16,6 +15,7 @@ describe('lib/services/messenger/translators/discourse.ts', () => {
 				hidden: false,
 				tags: [],
 				text: 'cde @test',
+				time: '2018-04-16T12:45:46+00:00',
```

Comment:
Quick question on time here. Do timestamps get translated? How are these handled for timezone, or do they all get converted to Zulu?

Reply:
In Jellyfish these are stored as an ISO string
In the Jellyfish translator in intend on doing a `moment(time).toISOString()`
Therefore in the common message type these should be in any format that momentjs can parse.
I'm adding this to the interface declaration.